helix-core/src/syntax.rs

```diff
@@ -881,6 +887,7 @@ impl TextObjectQuery {
                 nodes.into_iter().map(CapturedNode::Single).next()
             }
         });
+        log::info!("tree-sitter textobject query took: {} us", query_time.elapsed().as_micros());
```

Comment:
This one here only looks at the time it takes to create the iterator. This will need to move into wherever this function is called in order to report how long it takes to run the textobjects.scm queries